catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -462,6 +462,7 @@ const WorkDetails: FunctionComponent<Props> = ({ work, itemUrl }: Props) => {
                 languages: [],
                 subjectsLabel: null,
                 genresLabel: null,
+                contributorsLabel: null,
```

Comment:
Just for consistency - could we please call this
```suggestion
                contributorsAgentLabel: null,
```